service-front/module/Application/src/Controller/Authenticated/Lpa/HowPrimaryAttorneysMakeDecisionController.php

```diff
@@ -31,7 +31,7 @@ public function indexAction()
             $postData = $this->request->getPost();
 
             if ($postData['how'] != PrimaryAttorneyDecisions::LPA_DECISION_HOW_DEPENDS) {
-                $form->setValidationGroup('how');
+                $form->setValidationGroup(array('how'));
```

Comment:
Can we change this to `[]` instead of `array()` so it's consistent with the rest of the code base. As we haven't used this pattern anywhere else in the code base.

Reply:
Done